stroeer/core/v1/article.proto

```diff
@@ -181,6 +181,17 @@ message Article {
   */
   repeated int64 onwards = 10;
 
+  /**
+  * ## `entities`
+  * ```protobuf
+  * repeated string entities = 100;
+  * ```
+  * Extracted entities from the article body like people, places, organizations etc.
+  *
+  * `entities` are deprecated and should not be used by clients, use `keywords` instead.
+  */
+  repeated string entities = 100 [deprecated = true];
```

Comment:
alternatively, we could add something like `source` to the new `Keyword` message and transport old and new keywords/entities with this message

Reply:
If we delete this field entry in the near future, then it's ok. Otherwise, I would remove the `deprecated` flag.

Reply:
do we still need the old entities now that we have the new ones from retresco?

Reply:
yes, legacy systems e.g. for tracking, themepages and content analystics need both for a transistion period